### PR TITLE
Support swift 5.0 on cocoapods

### PR DESCRIPTION
**Pull request by bugix**
Support swift 5.0 on cocoapods

### CODE REVIEW

File: Sources/ZIPFoundation/Entry.swift [hunk 1]
```diff
@@ -11,7 +11,7 @@
 import Foundation
 import CoreFoundation
 
-/// A value that represents a file, a direcotry or a symbolic link within a ZIP `Archive`.
+/// A value that represents a file, a directory or a symbolic link within a ZIP `Archive`.
```
Comment (weichsel) [inline Sources/ZIPFoundation/Entry.swift]: Good catch - Thanks!

File: ZIPFoundation.podspec [hunk 2]
```diff
@@ -7,12 +7,12 @@ Pod::Spec.new do |s|
   s.social_media_url = 'http://twitter.com/weichsel'
   s.authors = { 'Thomas Zoechling' => 'thomas@peakstep.com' }
   s.source = { :git => 'https://github.com/weichsel/ZIPFoundation.git', :tag => s.version }
-  s.swift_version = '4.0'
+  s.swift_versions = ['4.0', '5.0']
```
Comment (weichsel) [inline ZIPFoundation.podspec]: As we also have some code paths for `SWIFT_VERSION = 4.2`, we maybe should also include `4.2` here.  
(I assume CocoaPods just passed this value to the compiler.  
As far as I know `SWIFT_VERSION` accepts the following values: `4.0`, `4.2` and `5.0`.

Comment (bugix) [reply ZIPFoundation.podspec]: Jup, let's add 4.2 as well

Comment (weichsel) [reply ZIPFoundation.podspec]: Thank you!

**Pull request merged**